roles/reproducer/tasks/main.yml

```diff
@@ -148,5 +149,6 @@
           ansible-playbook deploy-edpm.yml \
             -e @scenarios/centos-9/base.yml \
             -e @scenarios/centos-9/edpm_ci.yml \
-            -e cifmw_openshift_password=12345678 $@
+            -e cifmw_openshift_password="{{ cifmw_openshift_password }}" \
```

Comment:
molecule is complaining about `undefined variable`

Reply:
need some more testing, I'll work on that on Monday